File: HashmatTheBraveWarrior.java
import java.util.Scanner;
import java.lang.Math;

public class HashmatTheBraveWarrior {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        long sizeOfHashmatsArmy, sizeOfOpponentsArmy, absoluteDifference;
        while (scanner.hasNextLong()) {
            sizeOfHashmatsArmy = scanner.nextLong();
            sizeOfOpponentsArmy = scanner.nextLong();
            absoluteDifference = Math.abs(sizeOfHashmatsArmy - sizeOfOpponentsArmy);
            System.out.println(absoluteDifference);
        }
        scanner.close();
    }
}

File: HashmatTheBraveWarrior.cpp
#include <iostream>
#include <cmath>

int main() {
    long sizeOfHashmatsArmy, opponentsArmySize, absoluteDifference;
    while (std::cin >> sizeOfHashmatsArmy >> opponentsArmySize) {
        absoluteDifference = std::abs(sizeOfHashmatsArmy - opponentsArmySize);
        std::cout << absoluteDifference << std::endl;
    }
    return 0;
}
